### PR TITLE
Add description  of ignore-checking-items

### PR DESCRIPTION
**Pull request by sunzhaoyang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by sunzhaoyang** [comment]
cc @Ehco1996 @GMHDBJD

File: dm/task-configuration-file-full.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ online-ddl: true                # 支持上游 "gh-ost" 、"pt" 的自动处理
 online-ddl-scheme: "gh-ost"     # `online-ddl-scheme` 已被弃用，建议使用 `online-ddl`。
 clean-dump-file: true           # 是否清理 dump 阶段产生的文件，包括 metadata 文件、建库建表 SQL 文件以及数据导入 SQL 文件
 collation_compatible: "loose"   # 同步 CREATE 语句中缺省 Collation 的方式，可选 "loose" 和 "strict"，默认为 "loose"。"loose" 模式不会显式补充上游缺省的 Collation，"strict" 会显式补充上游缺省的 Collation。当使用 "strict" 模式，但下游不支持上游缺省的 Collation 时，下游可能会报错。 
+ignore-checking-items: []       # 忽略检查项。可用值请参考 precheck 说明页面
```
Comment (sunzhaoyang) [inline dm/task-configuration-file-full.md:36]:
```suggestion
ignore-checking-items: []       # 忽略检查项。可用值请参考 precheck 说明页面。
```

**Comment by TomShawn** [comment]
@sunzhaoyang 这个是基于现有或以往版本进行文档补充，还是规划在 v6.0 中发布的新特性呢？

**Comment by sunzhaoyang** [comment]
> 

缺失文档的补充

File: dm/dm-precheck.md [hunk 2]
```diff
@@ -110,6 +110,25 @@ tiup dmctl check-task ./task.yaml
 
 对于全量加增量数据迁移模式（`task-mode: all`），除了[通用检查项](#通用检查项)，前置检查还会包含[全量数据迁移检查项](#全量数据迁移检查项)，以及[增量数据迁移检查项](#增量数据迁移检查项)。
 
+### 可忽略检查项
+
+一般情况下前置检查项用于提前发现环境中可能存在的风险，不建议忽略。但考虑到可能存在某些特殊场景，仍然支持通过配置跳过部分检查项。
```
Comment (ran-huang) [inline dm/dm-precheck.md:115]:
```suggestion
一般情况下前置检查项用于提前发现环境中可能存在的风险，不建议忽略。如果你的数据迁移任务面临特殊场景，你可以通过 [`ignore-checking-items` 配置项](/dm/task-configuration-file-full.md#完整配置文件示例)来跳过部分检查项。
```

File: dm/dm-precheck.md [hunk 3]
```diff
@@ -110,6 +110,25 @@ tiup dmctl check-task ./task.yaml
...
+|version|检查上游数据库版本|
+|server_id|检查上游数据库是否设置 server_id|
+|binlog_enable|检查上游数据库是否已启用 binlog|
+|table_schema|检查上游 MySQL 表结构的兼容性|
+|schema_of_shard_tables|检查上游 MySQL 多实例分库分表的表结构一致性|
+|auto_increment_ID|关闭检查上游 MySQL 多实例分库分表的自增主键冲突|
+
+> **注意：**
+>
+> 旧版本支持忽略更多的检查项，但已经发现例如上游的`binlog_row_image`参数配置错误的情况下会导致同步时丢失数据，因此在新版本中移除了部分与数据安全相关的检查项。
```
Comment (sunzhaoyang) [inline dm/dm-precheck.md:130]:
```suggestion
> 6.0 之前的版本支持忽略更多的检查项，但诸如`binlog_row_image`等参数，若配置错误可能导致同步时丢失数据，因此在 6.0 版本中移除了部分与数据安全相关的检查项。
```

File: dm/dm-precheck.md [hunk 4]
```diff
@@ -110,6 +110,25 @@ tiup dmctl check-task ./task.yaml
...
+|version|检查上游数据库版本|
+|server_id|检查上游数据库是否设置 server_id|
+|binlog_enable|检查上游数据库是否已启用 binlog|
+|table_schema|检查上游 MySQL 表结构的兼容性|
+|schema_of_shard_tables|检查上游 MySQL 多实例分库分表的表结构一致性|
+|auto_increment_ID|关闭检查上游 MySQL 多实例分库分表的自增主键冲突|
+
+> **注意：**
+>
+> 6.0 之前的版本支持忽略更多的检查项，但诸如`binlog_row_image`等参数，若配置错误可能导致同步时丢失数据，因此在 6.0 版本中移除了部分与数据安全相关的检查项。
```
Comment (ran-huang) [inline dm/dm-precheck.md:130]:
```suggestion
> 6.0 之前的版本支持忽略更多的检查项，但诸如 `binlog_row_image` 等参数，若配置错误可能导致同步时丢失数据，因此在 6.0 版本中移除了部分与数据安全相关的检查项。
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

**Review by Ehco1996 — APPROVED** [review]
LGTM 
but FYI ,we are adding more check items in https://github.com/pingcap/tiflow/pull/5185

**Comment by sunzhaoyang** [comment]
> LGTM
> but FYI ,we are adding more check items in [pingcap/tiflow#5185](https://github.com/pingcap/tiflow/pull/5185)

We have to update this doc after the pr released.

**Pull request merged**